test/__mocks__/text.mock.ts

```diff
@@ -253,6 +253,7 @@ export const UPDATE_MANAGE_TRUSTS_REVIEWED_HEADING = "What you have reviewed";
 
 // Remove journey
 export const REMOVE_SOLD_ALL_LAND_FILTER_PAGE_TITLE = "Has the overseas entity disposed of all its property or land in the UK?";
```

Comment:
is REMOVE_SOLD_ALL_LAND_FILTER_PAGE_TITLE still used anywhere? if not, it can be removed and the "NEW" one can be renamed to remove the "NEW" from it.

Reply:
looks like those 2 pages are still showing the old page title, perhaps they need updating too. I would check with Vernon if they should be updated as well.

Reply:
They do, but yesterday in refinement was advised to leave the additional work and it'll be placed in the backlog.

Reply:
Another ticket has been created to amend the const that is used on both update.review.statement and check.your.answers